src/storage/initialUsers.ts

```diff
@@ -0,0 +1,26 @@
+import { User } from '../types/user.js';
+
+
+export const initialUsers: User[] = [
```

Comment:
instead of storage like this, I would recommend to have the seeder mechanism within the `sequalize` ORM.
I.e. we could have `data-access/migrations` and `data-access/seeders` and place there all required data.